types/stylelint/index.d.ts

```diff
@@ -849,6 +865,13 @@ declare namespace stylelint {
 		column: number;
 	};
 
+	export type FixCallback = () => void | undefined | never;
+
+	export type FixObject = {
+		apply?: FixCallback;
```

Comment:
question

Since `fix` can also be a function and that `Function.prototype.apply` exists, it could be safer to name this `mutate`?
In practice it shouldn't matter though; I am not sure.
@ybiquitous what's your opinion?